docs/_posts/2018-02-16-advanced-search.md

```diff
@@ -0,0 +1,240 @@
+<td name='libraries'>🔵</td>
+<td name='dilutions'>🔵</td>
+<td name='pool'>🔵</td>
+<td name='run'>🔵</td>
+<td name='container'>🔵</td>
+<td name='ordercomp'></td>
+</tr>
+
+<tr>
+  <td name='syntax'>changed:<a href="#dates">DATE</a><br/>modified:<a href="#dates">DATE</a><br/>updated:<a href="#dates">DATE</a><br/>changedon:<a href="#dates">DATE</a><br/>modifiedon:<a href="#dates">DATE</a><br/>updatedon:<a href="#dates">DATE</a></td>
+<td name='meaning'>Checks when any person last edited this item.</td>
+<td name='samples'>🔵</td>
+<td name='libraries'>🔵</td>
+<td name='dilutions'>🔵</td>
+<td name='pool'>🔵</td>
+<td name='run'>🔵</td>
+<td name='container'>🔵</td>
+<td name='ordercomp'>🔵</td>
+</tr>
+
+<tr>
+  <td name='syntax'>received:<a href="#dates">DATE</a><br/>recieved:<a href="#dates">DATE</a><br/>receivedon:<a href="#dates">DATE</a><br/>recievedon:<a href="#dates">DATE</a></td>
+  <td name='meaning'>Checks whether this item has a received date that matches the provided date.</td>
+  <td name='samples'>🔵</td>
+  <td name='libraries'></td>
+  <td name='dilutions'></td>
+  <td name='pool'></td>
+  <td name='run'></td>
+  <td name='container'></td>
+  <td name='ordercomp'></td>
+</tr>
+
+<tr>
+  <td name='syntax'>createdby:<a href="#user">USER</a><br/>creator:<a href="#user">USER</a><br/>creater:<a href="#user">USER</a></td>
+  <td name='meaning'>Check for items entered into MISO by a particular user.</td>
+  <td name='samples'>🔵</td>
+  <td name='libraries'>🔵</td>
+  <td name='dilutions'>🔵</td>
+  <td name='pool'>🔵</td>
+  <td name='run'>🔵</td>
+  <td name='container'>🔵</td>
+  <td name='ordercomp'></td>
+</tr>
+<tr>
+  <td name='syntax'>changedby:<a href="#user">USER</a><br/>modifier:<a href="#user">USER</a><br/>updater:<a href="#user">USER</a></td>
+  <td name='meaning'>Checks for the last person to edit this item in MISO.</td>
+  <td name='samples'>🔵</td>
+  <td name='libraries'>🔵</td>
+  <td name='dilutions'>🔵</td>
+  <td name='pool'>🔵</td>
+  <td name='run'>🔵</td>
+  <td name='container'>🔵</td>
+  <td name='ordercomp'></td>
+</tr>
+<tr>
+  <td name='syntax'>platform:PLATFORM</td>
+  <td name='meaning'>Check if this item is meant for a particular platform type: ILLUMINA, LS454, SOLID, IONTORRENT, PACBIO, OXFORDNANOPORE.</td>
+  <td name='samples'></td>
+  <td name='libraries'>🔵</td>
+  <td name='dilutions'>🔵</td>
+  <td name='pool'>🔵</td>
+  <td name='run'>🔵</td>
+  <td name='container'>🔵</td>
+  <td name='ordercomp'>🔵</td>
+</tr>
+<tr>
+  <td name='syntax'>index:<a href="#name">NAME</a><br/>index:SEQ</td>
+  <td name='meaning'>Checks if this item has the index provided. The index can be a name or a DNA sequence. The sequence must be an exact match.</td>
+  <td name='samples'></td>
+  <td name='libraries'>🔵</td>
+  <td name='dilutions'>🔵</td>
+  <td name='pool'>🔵</td>
+  <td name='run'>🔵</td>
+  <td name='container'>🔵</td>
+  <td name='ordercomp'>🔵</td>
+</tr>
+<tr>
+  <td name='syntax'>class:<a href="#name">NAME</a></td>
```

Comment:
Please highlight that the search terms `class`, `institute`, and `external` are all Detailed Sample only.

Reply:
I found a new emoji to signal this and put it into the legend.